examples/platforms/kw41z/openthread-core-kw41z-config.h

```diff
@@ -81,6 +81,6 @@
   * Define to 1 if you want use legacy transmit done.
   *
   */
-#define OPENTHREAD_CONFIG_LEGACY_TRANSMIT_DONE 1
+#define OPENTHREAD_CONFIG_LEGACY_TRANSMIT_DONE 0
```

Comment:
Here, the `OPENTHREAD_CONFIG_LEGACY_TRANSMIT_DONE` config option should be removed. Since KW41Z won't need to use the legacy transmit done callback anymore.